stacks/1.41/Dockerfile

```diff
@@ -0,0 +1,55 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1.2.1
+# Software:         Stacks
+# Software Version: 1.41
+# Description:      Stacks was developed to work with restriction enzyme-based data, such as RAD-seq, for the purpose of building genetic maps and conducting population...
+# Website:          http://catchenlab.life.illinois.edu/stacks/
+# Tags:             Genomics
+# Provides:         Stacks 1.41
+# Base Image:       biodckr/biodocker:latest
+# Build Cmd:        docker build --rm -t biodckr/stacks 1.41/.
+# Pull Cmd:         docker pull biodckr/stacks
+# Run Cmd:          docker run --rm biodckr/stacks <tool> <options>
+#################################################################
+
+# Set the base image to biodckr/biodocker
+FROM biodckr/biodocker:latest
```

Comment:
same remarks, should use biocontainers base image and use LABEL for metadata (+ license info)